### PR TITLE
Various fixes to Prepare your first contract tutorial

### PR DESCRIPTION
**Pull request by bencgreenberg**
- Fixed typo
- Put link to releases page to find the latest release tag
- Made the command snippet clearer on the need to paste in one's own contract ID before running

### CODE REVIEW

**Review by lisa-parity — APPROVED** [review]

File: content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md [hunk 1]
```diff
@@ -406,7 +404,7 @@ Since we are only reading from the blockchain state (we're not writing any new d
 avoid submitting an extrinsic.
 
 ```bash
-cargo contract call --contract 5GRAVvuSXx8pCpRUDHzK6S1r2FjadahRQ6NEgAVooQ2bB8r5 --message get --suri //Alice --dry-run
+cargo contract call --contract PUT_CONTRACT_ADDRESS_HERE --message get --suri //Alice --dry-run
```
Comment (HCastano) [inline content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md:407]: Can you make a note reminding readers that this values comes from the `cargo contract instantiate` step from earlier? 

```suggestion
cargo contract call --contract $INSTANTIATED_CONTRACT_ADDRESS --message get --suri //Alice --dry-run
```

File: content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md [hunk 2]
```diff
@@ -441,7 +439,7 @@ blockchain.
 To do this we can use the following command:
 
 ```bash
-cargo contract call --contract 5GQwxP5VTVHwJaRpoQsK5Fzs5cERYBzYhgik8SX7VAnvvbZS --message flip --suri //Alice
+cargo contract call --contract PUT_CONTRACT_ADDRESS_HERE --message flip --suri //Alice
```
Comment (HCastano) [inline content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md:442]:
```suggestion
cargo contract call --contract $INSTANTIATED_CONTRACT_ADDRESS --message flip --suri //Alice
```

File: content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ To install the contracts node on macOS or Linux:
 
 1. Open the downloaded file and extract the contents to a working directory.
 
-If you can't download the precompiled node, you can compile it locally with a command similar to the following:
+If you can't download the precompiled node, you can compile it locally with a command similar to the following. You can find the latest tag on the [Releases](https://github.com/paritytech/substrate-contracts-node/releases) page:
```
Comment (HCastano) [inline content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md:107]:
```suggestion
If you can't download the precompiled node, you can compile it locally with a command similar to the following. 
You can find the latest tag on the [Releases](https://github.com/paritytech/substrate-contracts-node/releases) page:
```

Long line

**Review by HCastano — COMMENTED** [review]
Couple of small things

**Review by HCastano — APPROVED** [review]
Thanks!

**Pull request merged**